include/frame.h

```diff
@@ -42,10 +42,51 @@ struct Order {
 };
 
 struct Unit {
+  enum Flags : int64_t {
+    BeingConstructed    = 1ll <<  4,
```

Comment:
Uh, I actually had the whole list of is* flags in there and then deleted a couple of them after Gabriel provided the necessary context :)